### DISCUSSION TITLE
Cowrie Visualization Top URL Download Error

### DISCUSSION THREAD

**Discussion by d3athg0d** [telekom-security/tpotce#1029] [Q&A]
Hi,

i have updated kibana and tpotce all to the latest version, however am still facing the same issue on the dashboard

Bucket
Field
url.keyword <== invalid

how can i rectify this?

![Screenshot 2022-02-27 043320](https://user-images.githubusercontent.com/78822442/155858249-0869eb81-5ada-436a-9c9c-049696ccb644.png)

**Comment by t3chn0m4g3** [comment]
As long the field has not been populated it will not show and throw the error. Check if the Cowrie log files in the data folder actually populated the field.
If the did follow the Wiki to install the latest Kibana objects.

**Comment by d3athg0d** [comment]
Hi,

the field url.keyword will populate by itself?

because it is not in the drop down list of the index pattern


![Screenshot 2022-02-27 052754](https://user-images.githubusercontent.com/78822442/155859638-19e2a678-4a5e-4a91-9719-37920836e498.png)
![Untitled](https://user-images.githubusercontent.com/78822442/155859640-c5a0fff5-3cd4-4907-b041-55042a632c9d.png)

**Reply by t3chn0m4g3** [reply]
Yes.
